README.md

```diff
@@ -78,37 +78,70 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Imagine you want to train a `LinearRegression` model based on your dataset `X`, `y`. After splitting the data into train/test segments you would have `X_train` and `y_train` as your training data. So you train your model as follows.
```

Comment:
- `X`, `Y`
- `Y_train`
- `model.fit(X_train, Y_train)`

Reply:
Since `y` is usually one-column and `X` is multi-column in learning setting, using small `y` and big `X` are conventional.